Modules/xASL_module_Structural.m

```diff
@@ -218,22 +234,29 @@
 
 %% -----------------------------------------------------------------------------
 %% 2    Register FLAIR -> T1w (if FLAIR.nii exists)
+% Additionally register T2 and T1c -> T1
 iState = 2;
 if ~x.mutex.HasState(StateName{iState}) % tracks progress through lock/*.status files, & locks current run
-    if xASL_exist(x.P.Path_FLAIR, 'file')
-
-        xASL_wrp_LinearReg_FLAIR2T1w(x, x.bAutoACPC);
+    if xASL_exist(x.P.Path_FLAIR, 'file') || xASL_exist(x.P.Path_T1c, 'file') || xASL_exist(x.P.Path_T2, 'file')
+		
+		if xASL_exist(x.P.Path_FLAIR, 'file')
+			xASL_wrp_LinearReg_FLAIR2T1w(x, x.bAutoACPC);
+		end
 
+		if xASL_exist(x.P.Path_T1c, 'file') || xASL_exist(x.P.Path_T2, 'file')
```

Comment:
Nice, I would also insert the other admin here (i.e. the folder creation for QC etc.), given how unlikely it is that this part will be run

Reply:
Done.